### PR TITLE
Fix sign up status message

### PR DESCRIPTION
**Pull request by rarira**
## Specify project
client

## Description

Related PR : #457

Some improvement and fix :

1. The "input-status" EditText of `SignUp` component should also have textInputProps.maxLength=60 props.
    By the way, Writing status while signup process looks bit weird, so it might be needless.

2. Use dooboo-ui `Snackbar` component for better UI of `Added to your friend` message
Simulator Screen Recording - iPhone 12 Pro - 2021-08-25 at 00 04 04

## Related Issues

N/A

## Tests

Updated ProfileModal test code: 
now checks if snackbar rendered

## Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/hackatalk/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] Run `yarn lint && yarn tsc`
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] I am willing to follow-up on review comments in a timely manner.

[wehack2021]-[cloud]

### CODE REVIEW

**Comment by rarira** [comment]
I'll make another commit, if https://github.com/dooboolab/hackatalk/pull/470 adds `Snackbar` to the Root

File: client/src/components/navigations/MainStackNavigator/ProfileModal.tsx [hunk 1]
```diff
@@ -468,10 +476,11 @@ const ModalContent: FC<ModalContentProps> = ({modalState, hideModal}) => {
           modalLayout={modalLayout}
           isStatusMessageExpanded={isStatusMessageExpanded}
           handleAnim={handleAnim}
-          showFriendAddedMessage={showFriendAddedMessage}
-          addFriendInFlight={addFriendInFlight}
         />
       )}
+      <View style={{zIndex: 101}}>
+        <Snackbar testID="profile-snackbar" theme={theme} ref={snackbarRef} />
+      </View>
```
Comment (hyochan) [inline client/src/components/navigations/MainStackNavigator/ProfileModal.tsx:483]: I've asked in #470 to put this in `RootStack` for reusability.

https://github.com/dooboolab/hackatalk/pull/470#discussion_r694929377

I think you might want to talk to @woosanggyu who is going to work on this :)

Comment (rarira) [reply client/src/components/navigations/MainStackNavigator/ProfileModal.tsx:483]: #470 is merged, and this PR is ready to be merged :)

**Review by hyochan — CHANGES_REQUESTED** [review]
I think you can continue this work after rebasing to `master`.

**Review by hyochan — APPROVED** [review]
LGTM~! Thanks for the great work again :1st_place_medal:

**Pull request merged**